PluginExecutionException

**Describe the bug**
While attempting to install dependencies, facing an execution error in stack-core dependency

I've executed those commands : 
```
mvn clean install
```

```
mvn dependency:tree
```

**Expected behavior**
Install depency simply and complete

**Logs and Packet Captures**
```
[INFO] stack-core ......................................... FAILURE [ 20.105 s]
```

```
[ERROR] Failed to execute goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest (generate-manifest) on project stack-core: Execution generate-manifest of goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest failed.: ConcurrentModificationException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest (generate-manifest) on project stack-core: Execution generate-manifest of goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest failed.
```
I've visited the apache wiki for this issue, told me to report the bug

**Additional context**
What told me ```mvn -version```
```
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: /usr/local/Cellar/maven/3.8.1/libexec
Java version: 15.0.2, vendor: N/A, runtime: /usr/local/Cellar/openjdk/15.0.2/libexec/openjdk.jdk/Contents/Home
Default locale: en_FR, platform encoding: UTF-8
OS name: "mac os x", version: "11.3", arch: "x86_64", family: "mac"
```

I'm using VSCode  and iTerm2 for CLI

Hope we'll find a solution :)

Indeed, I can reproduced this building with JDK 15 but not 8 or 11:

```
kevin@kevins-mbp ~/D/milo (master)> mvn -version
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: /usr/local/Cellar/maven/3.8.1/libexec
Java version: 15.0.2, vendor: AdoptOpenJDK, runtime: /Users/kevin/Library/Java/JavaVirtualMachines/adopt-openjdk-15.0.2/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"
kevin@kevins-mbp ~/D/milo (master)> java -version
openjdk version "15.0.2" 2021-01-19
OpenJDK Runtime Environment AdoptOpenJDK (build 15.0.2+7)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 15.0.2+7, mixed mode, sharing)
```

```
[ERROR] Failed to execute goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest (generate-manifest) on project stack-core: Execution generate-manifest of goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest failed.: ConcurrentModificationException -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :stack-core
```

It seems so :
```
java -version
```
Does this : 
```
openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (build 11.0.11+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (build 11.0.11+9, mixed mode)
```
And, for maven : 
```
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: /usr/local/Cellar/maven/3.8.1/libexec
Java version: 11.0.11, vendor: AdoptOpenJDK, runtime: /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
Default locale: en_FR, platform encoding: UTF-8
OS name: "mac os x", version: "11.3", arch: "x86_64", family: "mac"
```

It lead to the same error : 
```
[ERROR] Failed to execute goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest (generate-manifest) on project stack-core: Execution generate-manifest of goal org.apache.felix:maven-bundle-plugin:4.0.0:manifest failed.: ConcurrentModificationException -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -1,4 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021 the Eclipse Milo Authors
+  ~
+  ~ This program and the accompanying materials are made
+  ~ available under the terms of the Eclipse Public License 2.0
+  ~ which is available at https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  -->
+
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
@@ -317,7 +327,7 @@
                 <plugin>
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
-                    <version>4.0.0</version>
+                    <version>5.1.2</version>
                     <executions>
                         <execution>
                             <id>generate-manifest</id>

